src/generator/src/extensionConfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "beta": {
+    "name": "MicrosoftGraphBeta",
+    "version": "0.1.8-preview"
+  },
+  "v1.0": {
+    "name": "MicrosoftGraphV1.0",
+    "version": "0.1.8-preview"
+  }
+}
```

Comment:
`name` is the extension identifier in the Bicep compiler, used to identify the extension in Bicep templates and compiled ARM templates.